src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -102,6 +120,8 @@ public static String getRandomHydrateBreakMessageForPersonality(HydrateReminderP
             case CARING:
                 breakMessage = getRandomBreakMessage(HYDRATE_BREAK_CARING_TEXT_LIST);
                 break;
+            case ROMANTIC:
+                breakMessage =  getRandomBreakMessage(HYDRATE_BREAK_ROMANTIC_TEXT_LIST);
```

Comment:
Please remove the extra whitespace here after the `=`.

Reply:
Yeah, I've added it.